aioftp/pathio.py

```diff
@@ -763,19 +766,20 @@ class Lister(AbstractAsyncLister):
         else:
 
             if node.type == "file":
-
```

Comment:
This empty line should be here. Since whole project written with some code convensions.